### ISSUE TITLE
Can't remove blobs from custom model

### ISSUE THREAD

**Issue by oovm** [podgorskiy/StyleGAN_Blobless#1]
### Steps to Reproduce

download model from:

https://mega.nz/#!0JVxHQCD!C7ijBpRWNpcL_gubWFR-GTBDJTW1jXI6ThzSxwaw2aE

set resolution=512 here:

https://github.com/podgorskiy/StyleGAN_Blobless/blob/3bf6f225a493df42d262743e5d41cb91f1c5c33c/convertor.py#L57

use config following:

```yaml
MODEL:
  LATENT_SPACE_SIZE: 512
  LAYER_COUNT: 8
  MAX_CHANNEL_COUNT: 512
  START_CHANNEL_COUNT: 32
  DLATENT_AVG_BETA: 0.995
  MAPPING_LAYERS: 8
OUTPUT_DIR: .
```

---

The final result doesn't seem to make any difference, (at least I can't see), blobs still there.

![](https://user-images.githubusercontent.com/17541209/71543782-29c64a80-29b2-11ea-8ad2-beae5b019342.png)

Can you help me fix this model? Thanks!

**Comment by podgorskiy** [comment]
It turns out that process of blob formation depends a lot on a trained model.

The model that you've provided seems to generate spikes that cause blobs after 8x8 resolution, not 64x64. Also, it seems that blobs now are much less distinctive from the useful signal.

It might be because generating anime is much different from generating natural images, it requires sharp lines and relatively flat colors with almost zero texture. 

In the model that you've provided, it seems that blobs are mostly originated from channels 80 and 73 at resolution 8x8. I've created a branch https://github.com/podgorskiy/StyleGAN_Blobless/tree/asuka where I adjusted the code to work with your model.

The commit: https://github.com/podgorskiy/StyleGAN_Blobless/commit/c0cd2d0c3a3e85d4b3eb6f3b177d284673a1ee47

It's still a hack, so it won't work for all cases, but it seems to work most of the time.

![issue1](https://user-images.githubusercontent.com/3229783/71550760-aacb1500-29a5-11ea-94eb-e70fa0f18d41.png)
